components/eam/src/dynamics/se/stepon.F90

```diff
@@ -85,7 +85,7 @@ subroutine stepon_init(dyn_in, dyn_out )
   use cam_history,            only: register_vector_field
   use gravity_waves_sources,  only: gws_init
   use phys_control,           only: use_gw_front
-  use cam_history_support,    only: max_fieldname_len
+  use cam_history_support,    only: max_fieldname_len, add_hist_coord
```

Comment:
`add_hist_coord` does not look like it is used here. I suggest not including it in the `use` statement if it is not used in this routine.